corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/email-request.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
I hate to say this, but I don't think this should be added yet. Because this module is included in `base_main.js` and therefore included on all pages, it'll turn on strict mode everywhere, which greatly increases the testing burden for this PR. We recently had a P1 related to unexpected strict mode (there's a retrospective email thread March 21).

Reply:
Oh that is interesting. I thought the strict mode would be limited to only the code in `email-request.js`. I wasn't aware it would also be applied for the modules importing the script. I'll remove this commit.

Reply:
Commit has been removed.